README.md

```diff
@@ -12,11 +12,12 @@ This plugin uses the [Mongeez](https://github.com/mongeez/mongeez) library and c
 ## Installation
  * First add a dependency for the plugin in build.gradle
 
+Grails App >= 3.3.x use `v0.2.5` and <=3.2.x use `v0.2.4`
 ```
     buildscript {
         dependencies {
             ...
-            classpath "com.causecode.plugins:grails-mongeez:0.2.4"
+            classpath "com.causecode.plugins:grails-mongeez:$version"
```

Comment:
Please update changelog. If there is none, please add one.

Reply:
Done, it was not present.